core/templates/core/home_page.html

```diff
@@ -21,7 +21,7 @@
                             {% if article.editors_pick %}
                                 <div class="editors-pick"><div>Editors Pick</div></div>
                             {% endif %}
-                            {% image article.main_image original as background_image %}
+                            {% image article.main_image fill-2048x1365 as background_image %}
```

Comment:
Will this be too big for the article images? I'm assuming they will push the size of the row to be larger if the images are big enough to do the full fill?  Or am I mistaken?